MWC7K 2019 news article with results (#2133)
ppy/osu-wiki: news/2019-03-21-mwc-7k-2019-concludes.md

---
layout: post
title: "osu!mania 7K World Cup 2019 Concludes"
date: 2019-03-21 02:00:00 +0000
---

Last weekend, the final keys were pressed, and the osu!mania 7K World Cup 2019 came to its grand conclusion. Read on to find out more!

![](/wiki/shared/news/banners/MWC7K_2019_banner.jpg)

86 players. 24 countries. 1 prize.

> The past six weeks brought the greatest osu!mania 7K players centre-stage, in a clash of skill, wit and tenacity. Riding off the highly successful 2v2 format introduced last year, coupled with with stronger teams and a freshly minted mappool to match, this year’s world cup was all set for a one-way course to success.

Reviewer: ```suggestion
The past six weeks brought the greatest osu!mania 7K players centre-stage in a clash of skill, wit and tenacity. Riding off of the highly successful 2v2 format introduced last year and coupled with with stronger teams and a freshly minted mappool to match, this year’s World Cup was all set for a one-way course to success.
```

Without further ado, let's get to the results!

![](/wiki/Tournaments/MWC/2019_7K/img/podium.png)

**Team South Korea takes home the gold, defending their osu!mania 7K Champion title for the 3rd year in a row!**

> Led by veteran player [Jakads](https://osu.ppy.sh/users/259972) for the third year in a row, Team South Korea showed indomitable will and consistency throughout the tournament. Despite a minor setback in the Semifinals stage, they marched on to claim the championship title that they so rightfully deserve. A splendid performance - we expected no less.

Reviewer: ```suggestion
Led by veteran player [Jakads](https://osu.ppy.sh/users/259972) for the 3rd year in a row, Team South Korea showed indomitable will and consistency throughout the tournament. Despite a minor setback in the Semifinals stage, they marched on to claim the championship title that they so rightfully deserve. A splendid performance was given—we expected no less.
```
usually it would be "third", but you have "3rd" in the previous sentence and showing the numeral makes it more clear that the number is important. it's not a grammar rule but I feel like that makes sense lol

> Team South Korea will receive the grand prize - $150 per team member, profile badge, and the highly coveted "osu!mania Champion" user title for one year!

Reviewer: ```suggestion
Team South Korea will receive the grand prize: $150 per team member, profile badge, and the highly coveted "osu!mania Champion" user title for one year!
```

> **Team Philippines makes a surprising surge for silver, marking their first podium appearance since 2014**

Reviewer: ```suggestion
**Team Philippines makes a surprising surge for silver, marking their first podium appearance since 2014.**
```

> After their amazing run in 2018, Team Philippines soared to even greater heights this year with new leadership in the form of [Cielo Day](https://osu.ppy.sh/users/2722489). Backed by amazing performances not least from the likes of [Entozer](https://osu.ppy.sh/u/277044) and [- Kura -](https://osu.ppy.sh/u/11420405) , Team Philippines was a worthy contender for 2nd place for sure.

Reviewer: ```suggestion
After their amazing run in 2018, Team Philippines soared to even greater heights this year with new leadership in the form of [Cielo Day](https://osu.ppy.sh/users/2722489). Backed by amazing performances not least from the likes of [Entozer](https://osu.ppy.sh/users/277044) and [- Kura -](https://osu.ppy.sh/users/11420405), Team Philippines was a worthy contender for 2nd place for sure.
```

Team Philippines will receive a prize of $80 per team member, and a MWC 7K profile badge to commemorate their success.

**Team China walks away proudly with the bronze, matching their stellar performance from last year.**

> With an identical roster to last year (though with a minor change in leadership), Team Captain 	[\[Crz\]Satori](https://osu.ppy.sh/u/7082178) continued to uphold the high standards and ideals of their country. Their 3rd place victory was well deserved indeed.

Reviewer: ```suggestion
With an identical roster to last year (though with a minor change in leadership), Team Captain [\[Crz\]Satori](https://osu.ppy.sh/users/7082178) continues to uphold the high standards and ideals of their country. Their 3rd place victory was well deserved indeed.
```

Team China will receive a prize of $40 per team member, and a MWC 7K profile badge to commemorate their success.

--------

I'd like to take this opportunity to thank all the players for their excellent sportsmanship, and the tournament staff for their undying commitment to making things run smoothly. Finally, I'd also like to thank the viewers for showing their support for the teams and creating a truly wonderful environment for this competition.

The conclusion of this World Cup also comes with a few farewells from certain notable individuals.

> First and foremost, HappyStick has announced his retirement from the World Cup management team. In one short year, his contributions to the official tournament scene have been groundbreaking, and quite frankly, simply too long to list out. He has set a precedent for future World Cups to come, and will be a shining beacon that we all aspire towards. He will be missed dearly.

Reviewer: ```suggestion
First and foremost, [HappyStick](https://osu.ppy.sh/users/256802) has announced his retirement from the World Cup management team. In one short year, his contributions to the official tournament scene have been groundbreaking, and quite frankly, simply too long to list out. He has set a precedent for future World Cups to come, and will be a shining beacon that we all aspire towards. He will be missed dearly.
```

> Just as importantly, this conclusion of this tournament marks the start of a new chapter for global \#1 player Jakads, having announced his retirement from the game in pursuit of future study at university. We wish him all the very best!

Reviewer: ```suggestion
Just as importantly, this conclusion of this tournament marks the start of a new chapter for global #1 player [Jakads](https://osu.ppy.sh/users/259972), having announced his retirement from the game in pursuit of future study at university. We wish him all the very best!
```
